editor/code_editor.cpp

```diff
@@ -1059,7 +1059,7 @@ void CodeTextEditor::update_editor_settings() {
 		text_editor->set_line_length_guidelines(TypedArray<int>());
 	}
 
-	_update_font_ligatures();
+	_update_text_editor_theme();
```

Comment:
Pretty sure this is unnecessary. Or at least it should be inside the condition you removed below.